src/navigation/_CollapsingNavigationMenu.jsx

```diff
@@ -87,6 +87,9 @@ const CollapsingNavigationMenu = ({ selectedPath = undefined, menuItems, onSelec
   const [isNewSelectedPath, setIsNewSelectedPath] = useState(false);
   const [previousSelectedPath, setPreviousSelectedPath] = useState(selectedPath);
   const [openKeys, setOpenKeys] = useState(openKeysToItem(menuItems[0], selectedPath));
+  const [currentNodeId, setCurrentNodeId] = useState();
+  const cursor = useRef(0);
```

Comment:
The cursor won't always start at index 0; if a user navigates directly to a routed link the cursor needs to start from there.

For example, navigating here and then using the arrow keys should start from the focused element:

https://terra-dev-si-collapsing-bn3drj.herokuapp.com/secondary-nav-test/terra-dev-site/secondary-scroll-test/filler/filler-test

Reply:
Updated the first `useEffect` to set cursor position to currently selected menu item.